src/online-members/sse-events.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+// setupDragDropEvent('.members-list');
+
+let currentOnlineList = [];
+let previousOnlineList = [];
+
+function setUpOnlineMembersEventSource(url) {
+  const eventSource = new EventSource(url, { withCredentials: true });
+
+  eventSource.onopen = (e) => {
+    console.info('The connection has been established.');
+  };
+
+  eventSource.onmessage = function (event) {
+    const objectData = JSON.parse(event.data);
+    currentOnlineList = objectData.users;
+
+    previousOnlineList.forEach((member) => {
+      members[member].isOnline = false;
+    });
+
+    currentOnlineList.forEach((member) => {
+      members[member].isOnline = true;
+    });
+
+    previousOnlineList = currentOnlineList;
+    updateMembersOnlineStatus(currentOnlineList);
+  };
+
+  eventSource.onerror = (e) => {
+    console.log('An error occurred while attempting to connect.', e);
+    eventSource.close();
+  };
+
+  return eventSource;
+}
+
+const onlineMembersEventSource =
+  setUpOnlineMembersEventSource(RDS_SSE_ONLINE_URL);
+
+function updateMembersOnlineStatus(onlieMembersList) {
+  let toPositionOfOnlineMember = 0;
```

Comment:
What does this variable do?

Reply:
it tracks the position of online members 
renaming this to `targetPositionOfOnlineMembers`